js/utils/get_events.js

```diff
@@ -0,0 +1,43 @@
+// Initialize variables
+var editor = document.getElementById('editor');
```

Comment:
@code-master-ajay This I believe is a stale code and not part of the feature being added in this PR.

Reply:
@bot2x ,yes it is not a part of the line numbers feature.